spyder/plugins/console/plugin.py

```diff
@@ -131,17 +133,12 @@ def get_sys_path(self):
         """
         return self.get_widget().get_sys_path()
 
-    def handle_exception(self, text, is_traceback, is_pyls_error=False,
-                         is_faulthandler_report=False):
+    @Slot(dict)
+    def handle_exception(self, error_data):
         """
         Handle any exception that occurs during Spyder usage.

```

Comment:
Please complete this docstring to include the interface of the `error_data`.

Please also follow https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Fixed